feature/experimental/qos/ate_tests/qos_ecn_config_test/qos_ecn_config_test.go

```diff
@@ -54,11 +54,61 @@ func TestMain(m *testing.M) {
 //     - https://github.com/karimra/gnmic/blob/main/README.md
 //
 
+const (
+	dropProfile               = "DropProfile"
+	forwardingGroup           = "fc"
+	schedulerMap              = "smap"
+	sequeneId                 = 1
```

Comment:
should be "sequenceID"
misspelling sequene->sequence and ID should follow the Go initialism rule (https://google.github.io/styleguide/go/decisions#initialisms)